proto/health.proto

```diff
@@ -1,20 +1,30 @@
 syntax = "proto3";
 
-package main;
+package proto;
 
-option go_package = "./proto";
+import "google/protobuf/wrappers.proto";
 
-service HealthServiceApi {
-    rpc GetAllTargets (EmptyRequest) returns (EmptyResponse) {}
-    rpc GetAllTargetsByMetadata (EmptyRequest) returns (EmptyResponse) {}
-    rpc GetAllMetricsByTargetType (EmptyRequest) returns (EmptyResponse) {}
+option go_package = "github.com/Merteg/pl-health-service";
+
+
+service HealthService {
+    rpc Register (RegisterRequest) returns (RegisterResponse) {}
+    rpc Push (PushRequest) returns (PushResponse) {}
+}
+
+message RegisterRequest {
+    repeated Target target = 1;
 }
 
-message EmptyRequest {
+message RegisterResponse {
     string empty = 1;
 }
 
-message EmptyResponse {
+message PushRequest {
+    repeated Health health = 1;
+}
+
+message PushResponse {
     string empty = 1;
```

Comment:
you can have a message with no fields so no need to add this empty param

Reply:
Thanks, I did not know that and also wondered if that was possible.